assets/js/components/utils/index.js

```diff
@@ -22,7 +24,7 @@ const getProductsRequests = ( { selected = [], search } ) => {
 	// If we have a large catalog, we might not get all selected products in the first page.
 	if ( isLargeCatalog && selected.length ) {
 		requests.push(
-			addQueryArgs( '/wc/blocks/products', {
+			addQueryArgs( NAMESPACE, {
```

Comment:
Good idea with using a constant here!  Probably not needed here, but if we find this string used in more files we may want to start a top-level `constants.js` file for constants used in multiple places.

Reply:
Agree, I didn't want to do it in this PR to avoid it growing indefinitely. How does it sound handling that it an follow-up?